packages/calypso-e2e/src/lib/pages/login-page.ts

```diff
@@ -61,4 +81,51 @@ export class LoginPage {
 		// Make sure after logging in that everything stablizes, so we can continue with the next action!
 		await this.page.waitForLoadState( 'load' );
 	}
+
+	/**
+	 * Navigates to the Login page.
+	 */
+	async visit(): Promise< void > {
+		await this.page.goto( getCalypsoURL( 'log-in' ), { waitUntil: 'networkidle' } );
+	}
+
+	/**
+	 * Clicks on a link to navigate to the account signup page.
+	 */
+	async clickSignup(): Promise< void > {
```

Comment:
When I first read the method name and header, I was surprised then by all the navigation that this method did. What do you think of renaming this to capture the fact that it does all the navigation and clicks the signup button. Something like `navigateAndStartSignup`?

Reply:
I can see how it is a bit awkward, though it's partially due to the in-between state of the `LoginPage`. I do plan on a large refactor to eliminate the use of either `LoginFlow` or `LoginPage`, because having both feels rather unnecessary now (and is a legacy of the way Selenium suite is set up). 

I would expect the methods and names to be shuffled around when that time comes. Would this be acceptable until then?

And yes, I will definitely refactor this - not am empty promise 😛

Reply:
Of course, that sounds great! :)